### DISCUSSION TITLE
Panic alarm

### DISCUSSION THREAD

**Discussion by leonbezuidenhout** [ParadoxAlarmInterface/pai#374] [Q&A]
Is it possible to trigger a PANIC alarm using the software, i.e a button in Home Assistant that will trigger a panic alarm?

Similar to the Paradox physical panic button, or the software panic in Insight Gold?

It seems like an obvious feature, but I cant find any reference to it. Thanks

**Comment by gdwaterworth** [comment]
As far as I know it is just a switch, so you should be able to turn it off and on with a service call  in HA
![image](https://user-images.githubusercontent.com/87119523/218281253-02fb5375-ce0c-4bda-b20c-6338217e9441.png)

**Comment by PilgrimToHyperion** [comment]
Might be helpful @leonbezuidenhout [https://mybroadband.co.za/forum/threads/panic-button-with-paradox-ip150-pai-addon-in-home-assistant.1203369/post-30737659](url)

**Comment by jjreit** [comment]
@leonbezuidenhout  had you success with the panic alarm button? I have the same issue and don't know how I can do it.

@yozik04 do you have any advice how we can trigger an alarm out of Home Assistant (using IP150 and evo192)?
